README.md

```diff
@@ -51,7 +51,6 @@ _Modular_ means that all the building blocks can be easily replaceable.
    * [ ] Docker on Linux _(optional)_
    * [ ] SGX on Graphene _(optional)_
 1. Payment platform
-    * [x] **Payments with NGNT**
```

Comment:
think we should not remove this goal but update it ( to GLM )

Reply:
That was my default action, but https://github.com/golemfactory/yagna/pull/1013#discussion_r570317731
:sunglasses:

Reply:
yes, but I'd merge those lines to mention GLM explicitly here again (beside line 37)

Reply:
@pnowosie missed that comment, replied there :)

@tworec i think we should not merge "requirements", since they have been implemented step by step